src/Resources/views/CRUD/edit_orm_many_to_one.html.twig

```diff
@@ -73,14 +73,14 @@ file that was distributed with this source code.
             {% endif %}
 
             {% if sonata_admin.edit == 'list' and sonata_admin.field_description.associationadmin.hasRoute('delete') and sonata_admin.field_description.associationadmin.hasAccess('delete') and btn_delete %}
-                <a  href=""
+                <button
```

Comment:
hmmm it is unlikely to happen, but I think this could be considered BC break as someone may rely in this being an `a`.

Reply:
I'm not sure if our BC promise covers such html changes. Otherwise we can't change anything in our templates without BC breaks. Same for JS/CSS

I'm okay with the `button` change

Reply:
I'm okay too, but we should have better explanations in the Changelog.